server/routes/aws/SECURITYHUB_README.md

```
@@ -0,0 +1,65 @@
+
+To actively feed audits into this system:
+
+1. **Obtain your Webhook Details**
+   - Provide your Aurora App URL and append your destination: `https://<YOUR-DOMAIN>/api/v1/aws/securityhub/webhook/<YOUR_ORG_ID>`
+   - Map a generated API key into your specific tenant (`user_tokens` row where provider is `aws_securityhub`).
+2. **AWS API Destination Setup**
+   - Navigate to Amazon EventBridge -> **API Destinations**.
+   - Create a new connection and set your API Key under "Authorization header" (Header name: `x-api-key`).
+3. **AWS EventBridge Rule**
+   - Instruct EventBridge to capture audits utilizing a catch-all pattern matching: 
+     ```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)